snowpack/src/commands/build.ts

```diff
@@ -28,9 +28,15 @@ import {
   replaceExt,
 } from '../util';
 import {getInstallTargets, run as installRunner} from './install';
+import { EsmHmrEngine } from '../hmr-server-engine';
 
 const CONCURRENT_WORKERS = require('os').cpus().length;
 
+let hmrEngine: EsmHmrEngine | null = null;
+function getUseHmr(config: SnowpackConfig) {
```

Comment:
in a few places we use the phrase "is HMR" or `is HMR enabled`. 

`getIsHmrEnabled()` or even `isHmrEnabled()` would be nice for consistency.

Reply:
I use `getIsHmrEnabled` instead.